src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -414,15 +416,29 @@ private AdsDataProvider() {
   }
 
   @DataProvider
-  public static Object[][] kruxStandardSegment() {
+  public static Object[][] kruxSegments() {
     return new Object[][]{
         {
-            "glee",
-            "Rachel",
-            "adtest",
-            "SyntheticTests/Krux/Page_1",
-            "pqdapsy7l"
-        }
+            "J-RIfJI0",
+            new Page("vim", "Vim_Tips_Wiki"),
+            "[\"pqdapsy7l\"]", // Standard segment for visiting adtest before
+            new Page("adtest", "SyntheticTests/Krux/Page_1"),
+            "[\"o8l9bis26\",\"pqdapsy7l\"]", // Both standard and real-time segment for adtest
+        },
+        {
+            null,
+            new Page("adtest", "SyntheticTests/Krux/Page_1"),
+            "[\"o8l9bis26\"]",
+            new Page("glee", "Glee_TV_Show_Wiki"),
+            "[]", // No o8l9bis26 (real time segment for adtest, they don't traverse through wikis)
+        },
+        {
+            null,
+            new Page("vim", "Vim_Tips_Wiki"),
+            "[]", // No pqdapsy7l (standard segment for adtest)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)